source/extensions/filters/http/common/factory_base.h

```diff
@@ -55,6 +58,16 @@ class FactoryBase : public Server::Configuration::NamedHttpFilterConfigFactory {
   createRouteSpecificFilterConfigTyped(const RouteConfigProto&,
                                        Server::Configuration::ServerFactoryContext&,
                                        ProtobufMessage::ValidationVisitor&) {
+    if (Runtime::runtimeFeatureEnabled(
+            "envoy.reloadable_features.check_unsupported_typed_per_filter_config")) {
+      throw EnvoyException(fmt::format(
+          "The filter {} doesn't support virtual host-specific configurations", name()));
+    }
+    ENVOY_LOG(warn,
+              "The filter {} doesn't support virtual host-specific configurations. Set runtime "
+              "config `envoy.reloadable_features.check_unsupported_typed_per_filter_config` as "
+              "true to reject any invalid virtual-host specific configuration.",
+              name())
```

Comment:
The continuous integration test run detected a build breakage on this line due to missing ; at the end of the line.

Reply:
thanks

Reply:
/wait

Build still looks broken.  Did you implement a fix but didn't "git push" it out?